Assets/MRTK/SDK/Features/UX/Scripts/InteractiveElement/States/UnityEventDefinitions/InteractionStateActiveEvent.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License
+
+using UnityEngine.Events;
+
+namespace Microsoft.MixedReality.Toolkit.UI.Interaction
+{
+    /// <summary>
+    /// A Unity event with an Interaction State as event data.  The Interaction State represents the state that was activated or set on.
+    /// This event is used in the StateManager when a state is set on/activated.
+    /// </summary>
+    [System.Serializable]
+    public class InteractionStateActiveEvent : UnityEvent<InteractionState> { }
```

Comment:
It would be useful if the InteractionState event included both the InteractionState AND the object that is the target of this event (e.g. the object receiving the event) -- I often find I miss not having an event's target when working with mrtk.

Reply:
This request has been added to the to do list issue above